backend/algorithms/errors.json

```diff
@@ -7968,411 +7827,156 @@
         ],
         "broke at": "Index 1, Enrolment"
     },
-    "CVEN2303": {
+    "CVEN4050": {
         "condition": {
-            "original": "Prerequisites: (ENGG1300 AND ENGG2400) OR (CVEN1300 AND CVEN2301)<br/>(Pre-requisite only applicable to UG cohort taking this course).<br/><br/>",
-            "processed": "(ENGG1300 && ENGG2400) || (CVEN1300 && CVEN2301) (only applicable to UG cohort taking this course)"
+            "original": "Prerequisite: 132 UOCs needed to enrol into this course and BE Hons Programs only<br/><br/>",
+            "processed": "132UOC && (AEROAH || BINFAH || CEICAH || CEICDH || COMPBH || CVENAH || CVENBH || ELECAH || ELECCH || GMATDH || MANFBH || MECHAH || MINEAH || MTRNAH || PETRAH || SENGAH || SOLAAH || SOLABH || TELEAH)"
         },
         "tokens": [
             "(",
-            "(",
-            "ENGG1300",
-            "&&",
-            "ENGG2400",
-            ")",
-            "||",
-            "(",
-            "CVEN1300",
+            "132UOC",
             "&&",
-            "CVEN2301",
-            ")",
             "(",
-            "only",
-            "applicable",
-            "to",
-            "UG",
-            "cohort",
-            "taking",
-            "this",
-            "course",
+            "AEROAH",
+            "||",
+            "BINFAH",
+            "||",
+            "CEICAH",
+            "||",
+            "CEICDH",
+            "||",
+            "COMPBH",
+            "||",
+            "CVENAH",
+            "||",
+            "CVENBH",
+            "||",
+            "ELECAH",
+            "||",
+            "ELECCH",
+            "||",
+            "GMATDH",
+            "||",
+            "MANFBH",
+            "||",
+            "MECHAH",
+            "||",
+            "MINEAH",
+            "||",
+            "MTRNAH",
+            "||",
+            "PETRAH",
+            "||",
+            "SENGAH",
+            "||",
+            "SOLAAH",
+            "||",
+            "SOLABH",
+            "||",
+            "TELEAH",
             ")",
             ")"
         ],
-        "broke at": "Index 1, ("
+        "broke at": "Index 1, 132UOC"
```

Comment:
:(

Reply:
wait this always fails at index 1... there is something else afoot here...